plugins/services/src/js/pages/task-details/TaskFileViewer.js

```diff
@@ -13,7 +13,22 @@ import SearchLog from "../../components/SearchLog";
 import TaskDirectory from "../../structs/TaskDirectory";
 import TaskDirectoryActions from "../../events/TaskDirectoryActions";
 
-class TaskFileViewer extends React.Component {
+export function getNewRoute(routePath, params, path) {
+  const hasFilePathParam = routePath.indexOf(":filePath") !== -1;
+  if (!hasFilePathParam && routePath.endsWith("/")) {
```

Comment:
If you store `routePath.indexOf(":filePath") `, you also can store `routePath.endsWith("/")` 😀 (you know what I'd go for 😇)

Reply:
I respectfully disagree. endsWith explains in detail what it does, whereas indexOf !== -1 is not very explaining

Reply:
Well, you can use `routePath.includes(":filePath")` :trollface: